packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -391,10 +393,7 @@ function TableVirtualizer({layout, collection, focusedKey, renderView, renderWra
   }, [bodyRef]);
 
   let onVisibleRectChange = useCallback((rect: Rect) => {
-    // setting the table width will recalculate column widths which we only want to do once the virtualizer is done initializing
-    if (state.virtualizer.contentSize.height > 0) {
-      setTableWidth(rect.width);
-    }
+    setTableWidth(rect.width);
```

Comment:
Was this part of the spooky column code? Or an optimization?

Reply:
This was that bit of code that we looked at together when we were fixing the tests. I don't recall the exact reason, but we didn't need to wait for initialization. It's not related to the spooky column though. I'll try to find more of the reason for the change.